edge-functions/ab-testing-statsig/pages/_middleware.ts

```diff
@@ -0,0 +1,53 @@
+import { NextRequest, NextResponse } from 'next/server'
+import statsig from 'statsig-node'
```

Comment:
This can be confusing, can we add a note saying that `statsig-node` supports working on a V8 environment?